slack/web/classes/elements.py

```diff
@@ -1,478 +1,1042 @@
+                elif t == ConversationMultiSelectElement.type:
+                    return ConversationMultiSelectElement(**d)
+                elif t == ChannelSelectElement.type:
+                    return ChannelSelectElement(**d)
+                elif t == ChannelMultiSelectElement.type:
+                    return ChannelMultiSelectElement(**d)
+                elif t == PlainTextInputElement.type:
+                    return PlainTextInputElement(**d)
+                elif t == RadioButtonsElement.type:
+                    return RadioButtonsElement(**d)
+                elif t == CheckboxesElement.type:
+                    return CheckboxesElement(**d)
+                elif t == OverflowMenuElement.type:
+                    return OverflowMenuElement(**d)
+                elif t == DatePickerElement.type:
+                    return DatePickerElement(**d)
+                else:
+                    logger.warning(
+                        f"Unknown element detected and skipped ({block_element})"
+                    )
+                    return None
+            else:
+                logger.warning(
+                    f"Unknown element detected and skipped ({block_element})"
+                )
+                return None
+        elif isinstance(block_element, (TextObject, BlockElement)):
+            return block_element
+        else:
+            logger.warning(f"Unknown element detected and skipped ({block_element})")
+            return None
+
+    @classmethod
+    def parse_all(
+        cls, block_elements: List[Union[dict, "BlockElement"]]
+    ) -> List["BlockElement"]:
+        return [cls.parse(e) for e in block_elements or []]
+
+
+# -------------------------------------------------
+# Interactive Block Elements
+# -------------------------------------------------
 
 
 class InteractiveElement(BlockElement):
+    action_id_max_length = 255
+
     @property
     def attributes(self) -> Set[str]:
-        return super().attributes.union({"action_id"})
+        return super().attributes.union({"alt_text", "action_id"})
 
-    action_id_max_length = 255
+    def __init__(
+        self,
+        *,
+        action_id: Optional[str] = None,
+        type: Optional[str] = None,
+        subtype: Optional[str] = None,
+        **others: dict,
+    ):
+        if subtype:
+            self._subtype_warning()
+        super().__init__(type=type or subtype)
+        show_unknown_key_warning(__name__, others)
 
-    def __init__(self, *, action_id: str, subtype: str):
-        super().__init__(subtype=subtype)
         self.action_id = action_id
 
     @JsonValidator(
         f"action_id attribute cannot exceed {action_id_max_length} characters"
     )
-    def action_id_length(self):
-        return len(self.action_id) <= self.action_id_max_length
+    def validate_action_id_length(self):
+        return (
+            self.action_id is None or len(self.action_id) <= self.action_id_max_length
```

Comment:
Fied an error that may occur when `self.action_id` is None. Technically it's possible to set it as None when constructing messages or views so that we should support such use cases.